### ISSUE TITLE
[SI-ERROR-TRUE] 1.3.1 Info and Relationships - Presentational attributes used

### ISSUE THREAD

**Issue by cathysnider** [CuBoulder/express#970]
## Problem Description per Siteimprove

Presentational attributes such as 'border', 'align', or 'bgcolor' are used. CSS should be used for styling instead. It's important to review these issues and determine if they can be fixed by moving the presentational attributes to your CSS. It's not always practical to make these changes, but should be made when possible.

## Page Description

- A top level page with a video shortcode, which creates an iframe.
- A lower-level page with a video reveal block, which also creates an iframe. 
- A lower-level page with a Facebook Activity block, which also creates an iframe.

## What we're doing: 

The video shortcode, the Video Reveal block, the Facebook Activity and the Twitter block generate an iframe:

**Shortcode:**
< iframe src="//www.youtube.com/embed/etc" class="video-filter video-youtube vf-dqw4w9wgxcq" frameborder="0" allowfullscreen="true" id="fitvid985369" style="z-index: 2147483646;" >< / iframe >

**Video Reveal:**
< iframe class="" src="//www.youtube.com/embed/etc" frameborder="0" allowfullscreen="" id="fitvid926979" >< / iframe>

**Facebook Activity**
< iframe src="//www.facebook.com/plugins/likebox.php?href={path}" scrolling="no" frameborder="0" style="border: none; overflow: hidden; width: 100%; height: 580px; z-index: 2147483646; position: relative;" allowtransparency="true" >< / iframe >

**Twitter**
also has frameborder=0

I believe the problem is that 'frameborder='0'

## Solution

This attribute is no longer present in HTML5. Use CSS instead: < iframe style="border:0;" …  / >

## Note

Neither the Google Calendar or the Google Maps generates this error. That code does not have a frameborder.
